src/connectService.ts

```diff
@@ -198,9 +199,9 @@ class ConnectService extends IframeService {
   async authCreate(accountId: string): Promise<Uint8Array> {
     const message = 'Add this account as a Ceramic authentication method'
     const authSecret = await this.authenticate(message)
-    const entropy = sha256(authSecret.slice(2))
-    this.storeAccount(accountId, entropy)
-    return Uint8Array.from(Buffer.from(entropy, 'hex'))
+    const entropy = hash(fromString(authSecret.slice(2), 'base16'))
```

Comment:
actually we i think we can/should remove hashing here now, the authsecrets returned from linking/auth implementations now return sha256(val) (and we could more strictly guarantee that there), this will likely cause confusion later

Reply:
probably makes sense for us to change authenticate to return a uint8array authsecret now as well

Reply:
Seems more like a nice to have rather than necessary. I don't see a harm in hashing a second time.